core/app/models/spree/customer_return.rb

```diff
@@ -1,5 +1,6 @@
 module Spree
   class CustomerReturn < Spree::Base
+    include Spree::Core::NumberGenerator.new(prefix: 'CR', length: 9)
```

Comment:
I think I'll change it so including this module sets the config, but also exposes a class method so you can access the number generator.

For Spree orders you than can do code like `Spree::Order.number_generator.prefix = 'FOO'` if you wanted. 

As I said above from code level it looked not like configuration of these details *is* a feature of the public interface. I never saw a "monkey patch this method to configure something" a like configuration interface (nor I recommend it).